src/views/download/l10n.json

```diff
@@ -36,6 +36,9 @@
   "download.yesAnswer" : "Yes.",
   "download.onPhone": "Can I install Scratch on my Android phone?",
   "download.onPhoneAnswer" : "No. The current version of Scratch for Android only works on tablets.",
+  "download.howUpdateApp" : "How do I update the Scratch app?",
+  "download.howUpdateAppAnswerApp" : "Open the Google Play store and check for updates. If your installation is managed by school administrators, they will need to update the version and make the update available to managed devices.",
+  "download.howUpdateAppAnswerDownload" : "To update Scratch for {operatingsystem} from this page, download the latest version and install. To check which version you have, click the Scratch logo.",
```

Comment:
Could we clarify that we mean you should click the Scratch logo within the desktop app, and not on the website?